### PR TITLE
State unlocked blog post

### PR DESCRIPTION
**Pull request by sjwiesman**
Blog post covering schema evolution and general state best practices. 

cc @fhueske

### CODE REVIEW

File: _posts/2019-12-10-state-unlocked-interacting-with-state-in-apache-flink.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+
+With all  the exciting new functionality added in Flink 1.9 as well as some solid ideas and discussions around bringing state in Flink the next level, the community is committed to making state in Apache Flink a fundamental element of the framework, something that is ever-present across versions and upgrades of your application and a component that is a true first-class citizen in Apache Flink. We encourage you to sign up to the [Mailing list](https://flink.apache.org/community.html) and stay on top of the announcements and new features in upcoming releases.
```
Comment (fhueske) [inline _posts/2019-12-10-state-unlocked-interacting-with-state-in-apache-flink.md]: bringing state in Flink the next level -> bringing state in Flink **to** the next level

Comment (fhueske) [reply _posts/2019-12-10-state-unlocked-interacting-with-state-in-apache-flink.md]: sign up to the Mailing list -> sign up to the mailing list

File: _posts/2019-12-10-state-unlocked-interacting-with-state-in-apache-flink.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+A savepoint is a distributed snapshot of the global state of an application at a logical point-in-time and is stored in an external distributed file system or blob storage such as HDFS, or S3. Upon upgrading an application or implementing a code change  — such as adding a new operator or changing a field — the Flink job can restart by re-loading the application state from the savepoint into the state backend, making it local and available for the computation and continue processing as if nothing had ever happened.
+
+<center>
+<img src="{{ site.baseurl }}/img/blog/2019-12-10-state-unlocked-interacting-with-state-in-apache-flink/managing-state-in-flink-visual-2.png" width="600px" alt="State in Apache Flink"/>
+</center>
+<br>
+
+<div class="alert alert-info">
+ :information_source: **What to do with Backpressure?**
```
[the remark this answers is not in the document]
Comment (sjwiesman) [reply _posts/2019-12-10-state-unlocked-interacting-with-state-in-apache-flink.md]: whoops

**Comment by sjwiesman** [comment]
Thanks for taking a look @fhueske, that's what I get for opening a PR right as everyone's going on vacation ;)

For whatever reason I could not get the website to build, I'm getting weird c-linker errors locally so please let me know if this does not build or render properly. I also updated the dates for january 29th.

**Comment by fhueske** [comment]
Thanks for the fixes @sjwiesman!
Will merge this blog post :-)

**Pull request closed without merging**